packages/cli/lib/artifact-generator.js

```diff
@@ -40,7 +41,7 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
    */
   checkLoopBackProject() {
     if (this.shouldExit()) return false;
-    if (this.config.get('lbVersion') >= 400) {
+    if (semver.gte(this.config.get('lbVersion'), '4.0.0')) {
```

Comment:
I'm a bit confused. Here is what I read: `if (storedLbVersion >= '4.0.0') report error and exit`. Is it intentional?

BTW, maybe we should use `satisfies` to check against a range.

Reply:
Good catch! For some reason it worked when I ran it in a new scaffolded project.